src/main/java/redis/clients/jedis/Connection.java

```diff
@@ -38,6 +42,11 @@
   private SSLParameters sslParameters;
   private HostnameVerifier hostnameVerifier;
 
+  private boolean isUDSConnection(String host) {
```

Comment:
> What about just check if it is an ABS path?
Maybe it could also be a relative path to the CWD?



> It's easy to avoid the recheck each time. Keep a static ConcurrentHashMap of the string and whether it is 'ok'. On the first reference do the lookup, store the results, and keep going. On subsequent lookups, read from the hash map. If it's there, you're done.
> 
> If you are betting that the status of each of these hosts will change over time, start up a Timer which clears the map every so often, as determined by your level of paranoia. So every so often a new check will be done.

Seems a bit unnecessary as `connect` shouldn't happen that often in the code. I was looking for an alternative to check the supplied path. I don't care if the file exists because the UDS library already checks this and throws a proper exception (just realized that we're not testing this).